scripts/zones/Port_Jeuno/npcs/Red_Ghost.lua

```diff
@@ -27,12 +28,18 @@ end
 
 entity.onTrigger = function(player, npc)
     local wildcatJeuno = player:getCharVar("WildcatJeuno")
+    local fellowQuest = player:getCharVar("[Quest]Unlisted_Qualities")
```

Comment:
Are you comfortable putting the quests into the interaction framework? 
If not, I'm sure hook and I can take care of it.

Reply:
Beautiful! Feel free to ping my on discord if you ever run into trouble with it.